cromulent/model.py

```diff
@@ -223,11 +222,11 @@ def generate_id(self, what):
 
 		return self.base_url + what.__class__._uri_segment + "/" + str(slug)		
 
-	def toJSON(self, what):
+	def toJSON(self, what, done={}):
 		""" Serialize what, making sure of no infinite loops """
-		self.done = {}
-		out = what._toJSON(top=True)
-		self.done = {}
+		if not done:
```

Comment:
Oh. This may take care of it, but in that case I think it's clearer if the `def` line just has `None` as the default value.

Reply:
Yeah, I think it needs this ... and agree that =None is cleaner. Will fix.